exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java

```diff
@@ -29,141 +35,131 @@ public  Game(){
 	public String createRockQuestion(int index){
 		return "Rock Question " + index;

```

Comment:
What are your thoughts of this method? is it needed? if not, is there a way of adding this logic to the rest of the questions?